app/src/main/kotlin/at/bitfire/davdroid/resource/LocalEvent.kt

```diff
@@ -51,6 +50,18 @@ class LocalEvent: AndroidEvent, LocalResource<Event> {
             )
         }
 
+        /**
+         * Marks the event as not deleted
+         * @param eventID
+         */
+        fun markAsNotDeleted(provider: ContentProviderClient, account: Account, eventID: Long) {
+            provider.update(
+                ContentUris.withAppendedId(Events.CONTENT_URI, eventID).asSyncAdapter(account),
+                ContentValues(1).apply { put(Events.DELETED, 0) },
+                null,null
+            )
+        }
```

Comment:
The same concept of method already exists for contacts in `LocalAddress.resetDeleted()`.